### PR TITLE
Release 2.21.2

### PR DESCRIPTION
**Pull request by jodygarnett**
Draft blog post for 2.21.2 release

### CODE REVIEW

File: _posts/2022-10-24-geoserver-2-21-2-released.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+* ``[POST] /workspaces/<ws>/datastores/<cs>/reset``
+* ``[POST] /workspaces/<ws>/datastores/<ds>/featuretypes/<ft>/reset``
+
+  Resetting a featuretype will not overwrite any attribute selection / renaming / type conversion as this has been supplied by hand and not generated (That kind of information can be updated via REST API explicit PUT on the feature type resource.)
+
+* ``[POST] /workspaces/<ws>/coveragestores/<cs>/reset``
+* ``[POST] /workspaces/<ws>/coveragestores/<cs>/coverages/<c>/reset``
+
+To use consult REST API reference for [coveragestores](http://docs.geoserver.org/latest/en/api/#1.0.0/coveragestores.yaml) / [coverages](http://docs.geoserver.org/latest/en/api/#1.0.0/coverages.yaml]), and  [datastores](http://docs.geoserver.org/latest/en/api/#1.0.0/datastores.yaml) / [featuretypes](http://docs.geoserver.org/latest/en/api/#1.0.0/featuretypes.yaml).
```
Comment (bradh) [inline _posts/2022-10-24-geoserver-2-21-2-released.md:43]:
```suggestion
Consult the REST API reference for [coveragestores](http://docs.geoserver.org/latest/en/api/#1.0.0/coveragestores.yaml) / [coverages](http://docs.geoserver.org/latest/en/api/#1.0.0/coverages.yaml]), and  [datastores](http://docs.geoserver.org/latest/en/api/#1.0.0/datastores.yaml) / [featuretypes](http://docs.geoserver.org/latest/en/api/#1.0.0/featuretypes.yaml) for detailed usage information.
```

File: _posts/2022-10-24-geoserver-2-21-2-released.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+The built-in logging profiles have been updated as the date was being incorrectly logged:
+
+* If you have hand edited any of the built-in logging profiles you can fix the data format manually. Locate appender ``PatternLayout`` entries and correct the date formatting to ``%date{dd MMM HH:mm:ss}``.
+
+* If you have not modified any of the built-in logging profiles a quick way to update is to remove them from your GEOSERVER_DATA_DIRECTORY ``logs`` folder.
+  
+  The built-in logging profiles will be restored next time you change profiles or when the application starts up. 
+
+* If you never plan to customize the built-in loggig profiles use the system property ``UPDATE_BUILT_IN_LOGGING_PROFILES=true``. This setting will cause GeoServer to update the files when changing profiles or on application startup.
```
Comment (bradh) [inline _posts/2022-10-24-geoserver-2-21-2-released.md:59]:
```suggestion
* If you never plan to customize the built-in logging profiles use the system property ``UPDATE_BUILT_IN_LOGGING_PROFILES=true``. This setting will cause GeoServer to update the files when changing profiles or on application startup.
```

**Review by bradh — APPROVED** [review]
@jodygarnett Some editorial fixes. Can you check I haven't changed your intent?

**Comment by jodygarnett** [comment]
> @jodygarnett Some editorial fixes. Can you check I haven't changed your intent?

Thanks that is prefect

**Pull request merged**